/**:
  ros__parameters:
    output_frame: base_link
    has_static_tf_only: true
    input_topics: [
                    "/sensing/lidar/left/pointcloud_before_sync",
                    "/sensing/lidar/right/pointcloud_before_sync",
                    "/sensing/lidar/top/pointcloud_before_sync"
                ]
    max_queue_size: 5
    timeout_sec: 0.033
    input_offset: [0.0 ,0.0 ,0.0]
